### PR TITLE
Added most of world countries

### PR DESCRIPTION
**Pull request by diffusi-on**

### CODE REVIEW

**Comment by ronatskiy** [comment]
Great job! Thanks.

File: src/data/countries.js [hunk 1]
```diff
@@ -13,75 +13,249 @@ class CountryData {
...
+	new CountryData("Belgium", "Belgian", "BE", "BEL"),
+	new CountryData("Belize", "Belizean", "BZ", "BLZ"),
+	new CountryData("Benin", "Beninese", "BJ", "BEN"),
+	new CountryData("Bermuda", "Bermudian", "BM", "BMU"),
+	new CountryData("Bhutan", "Bhutanese", "BT", "BTN"),
+	new CountryData("Bolivia", "Bolivian", "BO", "BOL"),
+	new CountryData("Bonaire, Sint Eustatius and Saba", "Dutch", "BQ", "BES"),
+	new CountryData("Bosnia and Herzegovina", "Bosnian", "BA", "BIH"),
+	new CountryData("Botswana", "Batswana", "BW", "BWA"),
+	new CountryData("Brazil", "Brazilian", "BR", "BRA"),
```
Comment (ronatskiy) [inline src/data/countries.js:44]: This one and a few more flags URLs below are broken. To see that you can run `npm run update-docs` and review the updated readme.md file.

**Comment by diffusi-on** [comment]
@ronatskiy Fixed broken flags. should be ok now. please check

**Comment by diffusi-on** [comment]
@ronatskiy Can you please review and merge PR if it is ok ?

**Comment by ronatskiy** [comment]
@diffusi-on I can't merge it because of conflicts.

**Comment by diffusi-on** [comment]
@ronatskiy Should be ok now after last commit.

**Comment by ronatskiy** [comment]
@diffusi-on Thanks a lot 👍

**Comment by ronatskiy** [comment]
The new version is published https://www.npmjs.com/package/country-flags-svg/v/1.1.0

**Pull request merged**